openfl/component/collaborator/collaborator.py

```diff
@@ -133,6 +135,14 @@ def set_available_devices(self, cuda: Tuple[str] = ()):
 
     def run(self):
         """Run the collaborator."""
+        if LOG_WANDB:
+            wandb.init(project="my_project", entity="my_group", tags=["my_tags"],
+                       config={
+                           "num_clients": 4,
+                           "rounds": 100,
+                       },
+                       name=self.collaborator_name
```

Comment:
Same comment as above. Make project, entity, tags configurable. Use client count and rounds from existing plan variables.